data_steward/validation/participants/participant_validation_queries.py

```diff
@@ -1,6 +1,28 @@
 # Project imports
 from common import JINJA_ENV
 
+CREATE_NAME_COMPARISON_FUNCTION = JINJA_ENV.from_string("""
+    CREATE FUNCTION IF NOT EXISTS `{{project_id}}.{{drc_dataset_id}}.CompareName`(rdr_name string, ehr_name string)
```

Comment:
@jp3477 Add a docstring to this function about the functionality and limitations of this function, as it will be easier to get around these in the future in order to expand the scope.

Reply:
How does it look now?